packages/block-library/src/cover/deprecated.js

```diff
@@ -30,6 +30,9 @@ import {
 } from './shared';
 
 const blockAttributes = {
+	align: {
+		type: 'string',
+	},
```

Comment:
I'm not entirely sure this is the correct fix :thinking: 

The 4th deprecation actually uses a `supports` declaration for `align`:

https://github.com/WordPress/gutenberg/blob/8a399548d5208affe5186afb0218d4304c436306/packages/block-library/src/cover/deprecated.js#L419-L421

So maybe we need to copy _that_ to all more recent deprecations? :thinking: 

cc/ @gziolo since AFAICT you wrote that deprecation.

Reply:
Did I? It might be as well moving code 😅
The best way to verify is to copy the old version in the post and check if it properly converts.

Reply:
> I'm not entirely sure this is the correct fix 🤔
> 
> The 4th deprecation actually uses a `supports` declaration for `align`:
> 
> https://github.com/WordPress/gutenberg/blob/8a399548d5208affe5186afb0218d4304c436306/packages/block-library/src/cover/deprecated.js#L419-L421
> 
> So maybe we need to copy _that_ to all more recent deprecations? 🤔
> 
> cc/ @gziolo since AFAICT you wrote that deprecation.

Followed this advice in https://github.com/WordPress/gutenberg/pull/28796/commits/0183f5ec3c036d3e9e0ccc23f2f6bb7126ad2e1f. Had to regenerate the fixtures but it looks better now and works well for me! Thanks! ❤️